### PR TITLE
Update music-theory-spectrum.csl

### PR DESCRIPTION
**Pull request by damnation333**
Via request here: https://forums.zotero.org/discussion/comment/271245/#Comment_271245
@rmzelle Can you confirm I made the right changes? Doing this blind due to lack of testing abilities. Thanks

### CODE REVIEW

**Comment by adam3smith** [comment]
(not right; fixing in commit right now)

File: music-theory-spectrum.csl [hunk 1]
```diff
@@ -391,13 +391,13 @@
     <layout suffix="." delimiter="; ">
       <choose>
         <if position="ibid-with-locator">
-          <group delimiter=", ">
+          <group delimiter=" " prefix="(" suffix=")">
```
Comment (rmzelle) [inline music-theory-spectrum.csl]: @damnation333, I think the parentheses belong on the `<text macro="point-locators-subsequent"/>` to get "Ibid. (187-8)." Currently the "ibid" term would be wrapped as well.

I also don't think you need the parentheses on the "ibid" term below.

Comment (adam3smith) [inline music-theory-spectrum.csl]: please merge once test passes?

**Comment by rmzelle** [comment]
Squashed!

image
(https://flic.kr/p/db5MqE)

**Pull request merged**